### PR TITLE
feat(config): update/consolidate HomeSeer parameters for WD200 and WX300

### PR DESCRIPTION
**Pull request by jeffhatt**
Add parameters 5 and 6 to HomeSeer WD-200+
Cleanup and consolidate common parameters

### CODE REVIEW

**Comment by jeffhatt** [comment]
~~How do I handle parameters that are only available in certain FW versions?
Do I need to duplicate all of the parameters or will it append them (as I currently have it setup)?~~
Found a working example Zooz ZEN27

**Comment by AlCalzone** [comment]
> How do I handle parameters that are only available in certain FW versions?

I think you figured it out. The change looks correct.

File: packages/config/config/devices/0x000c/templates/homeseer_template.json [hunk 1]
```diff
@@ -105,6 +105,39 @@
 		"$import": "~/templates/master_template.json#dimming_timing",
 		"valueSize": 2
 	},
+	"dimming_threshold": {
+		"label": "Dimming Threshold",
+		"description": "Sets the lowest dimming threshold",
```
Comment (blhoward2) [inline packages/config/config/devices/0x000c/templates/homeseer_template.json]: This description is unnecessary.

File: packages/config/config/devices/0x000c/templates/homeseer_template.json [hunk 2]
```diff
@@ -105,6 +105,39 @@
...
+	},
+	"ramp_rate": {
+		"valueSize": 1,
+		"unit": "seconds",
+		"minValue": 0,
+		"maxValue": 90,
+		"defaultValue": 3
+	},
+	"scene_control": {
+		"label": "Central Scene, Multi-Tap, Press and Hold",
```
Comment (blhoward2) [inline packages/config/config/devices/0x000c/templates/homeseer_template.json]: Let's just make this label "Central Scene Commands"

**Review by blhoward2 — CHANGES_REQUESTED** [review]

File: packages/config/config/devices/0x000c/templates/homeseer_template.json (hunk 1 above)
Comment (jeffhatt) [reply packages/config/config/devices/0x000c/templates/homeseer_template.json]: Removed.

File: packages/config/config/devices/0x000c/templates/homeseer_template.json (hunk 2 above)
Comment (jeffhatt) [reply packages/config/config/devices/0x000c/templates/homeseer_template.json]: Updated.

**Review by blhoward2 — APPROVED** [review]

**Review by AlCalzone — APPROVED** [review]

**Pull request merged**